main.py

```diff
@@ -10,14 +10,50 @@
 NUSCENES_ROOT = 'nuscenes'
 
 SENSOR_NAMES = {
-    'LIDAR_TOP': 'lidar'
+    'LIDAR_TOP': 'lidar',
     'CAM_FRONT': 'ring_front_center',
     'CAM_FRONT_LEFT': 'ring_front_left',
     'CAM_FRONT_RIGHT': 'ring_front_right',
     'CAM_BACK_LEFT': 'ring_side_left',
     'CAM_BACK_RIGHT': 'ring_side_right',
 }
 
+def is_camera_sensor(sensor):
+    return sensor[0:3] == "CAM"
+
+def get_calibration_info(nusc, scene):
```

Comment:
thanks for this PR, Bhavya. Do you mind adding type hints to the input and return arguments to each function?

Here are a few pointers:
https://docs.python.org/3/library/typing.html
https://www.python.org/dev/peps/pep-0484/

Reply:
Added in the latest commit